app/core/Menu.js

```diff
@@ -117,7 +117,13 @@ var Menu = {
       width: 575,
       height: 270,
       shortcut: 'CmdOrCtrl+Shift+P'
-    }
+    },
+    'Control Window':{
+      openWindow: 'controlWindow',
+      width: 1000,
+      height: 700,
+      devTools: true,
```

Comment:
set this to false. You dont want dev tools to be displayed when a user opens up a window